README.md

```diff
@@ -1,52 +1,64 @@
-- [Crescendo.Preview.4](https://devblogs.microsoft.com/powershell/announcing-powershell-crescendo-preview-4/)
+To install **Microsoft.PowerShell.Crescendo**:
 
-You can learn more about our approach from the blog discussion:
-[Part 1](https://devblogs.microsoft.com/powershell/native-commands-in-powershell-a-new-approach/)
-and
-[Part 2](https://devblogs.microsoft.com/powershell/native-commands-in-powershell-a-new-approach-part-2)
+```powershell
+Install-Module -Name Microsoft.PowerShell.Crescendo
+```
 
-## How to use Crescendo
+To install **Microsoft.PowerShell.Crescendo** using the new
+[PowerShellGet.v3](https://www.powershellgallery.com/packages/PowerShellGet/3.0.12-beta)
 
-For more information using **Microsoft.PowerShell.Crescendo**, check out this excellent blog series
-by Sean Wheeler posted to the
-[PowerShell Community](https://devblogs.microsoft.com/powershell-community/).
+```powershell
+Install-PSResource -Name Microsoft.PowerShell.Crescendo
+```
 
-* [My Crescendo journey](https://devblogs.microsoft.com/powershell-community/my-crescendo-journey/)
-* [Converting string output to objects](https://devblogs.microsoft.com/powershell-community/converting-string-output-to-objects/)
-* [A closer look at the parsing code of a Crescendo output handler](https://devblogs.microsoft.com/powershell-community/a-closer-look-at-the-parsing-code-of-a-crescendo-output-handler/)
-* [A closer look at the Crescendo configuration](https://devblogs.microsoft.com/powershell-community/a-closer-look-at-the-crescendo-configuration/)
+## Documentation and more information
 
-## Community guidance
+To get started using Crescendo, check out the
+[documentation](https://docs.microsoft.com/powershell/utility-modules/crescendo/overview).
 
-We are approaching our general release (GA) of **PowerShell Crescendo 1.0**. We look forward to
-community feedback and suggestions. Please file issues for erroneous behavior or suggested features.
-As we stabilize for GA, we will not be taking feature PRs.
+For a detailed walkthrough using Crescendo, see this excellent blog series from Sean Wheeler -
+Thanks Sean!
+
+- Crescendo on the [PowerShell Community Blog](https://devblogs.microsoft.com/powershell-community/tag/crescendo/).
+
+## Future plans
+
+We value your ideas and feedback and hope you will give Crescendo a try and let us know of any
+issues you find.
+
+## Release history
+
+- 12/20 - [Crescendo.Preview.1](https://devblogs.microsoft.com/powershell/announcing-powershell-crescendo-preview-1/)
+- 5/21 - [Crescendo.Preview.2](https://devblogs.microsoft.com/powershell/announcing-powershell-crescendo-preview-2/)
+- 7/21 - [Crescendo.Preview.3](https://devblogs.microsoft.com/powershell/announcing-powershell-crescendo-preview-3/)
+- 10/21 - [Crescendo.Preview.4](https://devblogs.microsoft.com/powershell/announcing-powershell-crescendo-preview-4/)
+- 12/21 - [Crescendo.RC](https://devblogs.microsoft.com/powershell/announcing-powershell-crescendo-release-candidate-rc/)
```

Comment:
```suggestion
Release announcements on the [PowerShell Blog](https://devblogs.microsoft.com/powershell/tag/powershell-crescendo/).

- 12/20 - Crescendo.Preview.1
- 05/21 - Crescendo.Preview.2
- 07/21 - Crescendo.Preview.3
- 10/21 - Crescendo.Preview.4
- 12/21 - Crescendo.RC
```

Reply:
@sdwheeler - ;) You did it!  That really works for me!